node/sc/bridge_test.go

```diff
@@ -222,7 +221,7 @@ loop:
 // TestBridgeHandleValueTransferNonceAndBlockNumber checks the following:
 // - the bridge allows the handle value transfer with only serialized nonce.
 // - the bridge correctly stores and returns the block number.
-func _TestBridgeHandleValueTransferNonceAndBlockNumber(t *testing.T) {
+func TestBridgeHandleValueTransferNonceAndBlockNumber(t *testing.T) {
```

Comment:
We need to modify the function name/comment. We don't need to test sequential nonce. :)

Reply:
Fixed. PTAL.